src/Akka.HealthCheck.Persistence/AkkaPersistenceLivenessProbe.cs

```diff
@@ -120,7 +104,7 @@ public AkkaPersistenceLivenessProbe(bool logInfo, TimeSpan delay, TimeSpan timeo
             _logInfo = logInfo;
             _log = Context.GetLogger();
             
-            Become(WarmingUp);
+            Become(Active);
```

Comment:
The warm-up state where the journal event and snapshot were written is removed, we go straight to using the very simple suicide actor that only checks for message recovery.